cpp/src/rolling/rolling.cu

```diff
@@ -1525,6 +1584,8 @@ std::unique_ptr<column> time_range_window_ASC(
     auto nulls_begin = d_nulls_begin[group_label];
     auto nulls_end   = d_nulls_end[group_label];
 
+    if (following_window_is_unbounded) { return group_end - idx - 1; }
```

Comment:
Nitpick:  should always err on the side of parentheses with compound arithmetic.